### ISSUE TITLE
Disallow protected on final classes

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#294]
I guess that it doesn't makes sense if you are not overring protected properties or methods.

```php
final class MyClass {
    protected $property;
}
```

It should warning "_$property could be private_".

**Comment by kalessil** [comment]
Nice finding, picking it up.

**Comment by rentalhost** [comment]
Fixed on #314, waiting for merging.
